src/crimson/os/seastore/onode_manager.h

```diff
@@ -34,18 +34,23 @@ class OnodeManager {
     Transaction &trans,
     const ghobject_t &hoid) = 0;
 
+  struct adjacent_onodes_t {
+    OnodeRef left;
+    OnodeRef onode;
+    OnodeRef right;
+  };
```

Comment:
This interface looks to be dropped, I suppose it was necessary because `OP_CREATE/TOUCH` may create a clone of an existing object, so we need to check the adjacent onodes. Is this assumption changed?

Also refer to https://github.com/ceph/ceph/pull/59476#discussion_r1820170090.

Reply:
This feature is only relevant to the recovery process, so I think moving it to a standalone PR can reduce the workload of review.